contents/docs/error-tracking/integrations.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Set up integrations
+---
+
+PostHog integrates with GitHub, GitLab, and Linear to enhance error tracking with source code context, commit information, and external issue tracking.
+
+## GitHub integration
+
+Connect your GitHub repository to enable:
+
+- [Source linking](/docs/error-tracking/stack-traces#source-linking) in stack traces
+- Commit details in error reports
+- External issue tracking
+
+To set up the GitHub integration:
+
+1. Go to [Error tracking settings](https://app.posthog.com/settings/project-error-tracking) in the PostHog app and find **Integrations**
+2. Under **GitHub**, click **Connect organization**
+3. Authorize PostHog to access your repositories
+4. Select the repositories you want to link
+
+PostHog connects to GitHub by installing a GitHub application. PostHog needs the following permissions:
+
+- Read access to metadata
+- Read and write access to code, issues, and pull requests
+
+Once connected, create a [release](/docs/error-tracking/releases) to start using source linking.
+
+## GitLab integration
+
+Connect your GitLab repository to enable:
+
+- [Source linking](/docs/error-tracking/stack-traces#source-linking) in stack traces
+- Commit details in error reports
+- External issue tracking
+
+To set up the GitLab integration:
+
+1. Go to [Error tracking settings](https://app.posthog.com/settings/project-error-tracking) in the PostHog app and find **Integrations**
+2. Under **GitLab**, click **Connect project**
+3. Provide the following information:
+   - Hostname
+   - Project ID
+   - A project access token with the `api` scope
+
+PostHog connects to GitLab using a [project access token](https://docs.gitlab.com/user/project/settings/project_access_tokens).
+
+Once connected, create a [release](/docs/error-tracking/releases) to start using source linking.
+
+## Linear integration
+
+Connect your Linear workspace to enable:
+
+- External issue tracking
```

Comment:
This is really weird as a list with 1 item... Dare I say, this can be a table? Then, the single item may not look as ugly.

See what looks better, I trust your taste more than mine.

 :sorry: I have a thing for tables.